nightfall/api.py

```diff
@@ -99,19 +136,29 @@ def scan(self, data):
             response = requests.post(
                 url=self._url,
                 headers=self._headers,
-                data=json.dumps(payload)
+                data=json.dumps(data)
             )
 
             # Logs for Debugging
             self.logger.debug(f"HTTP Request URL: {response.request.url}")
             self.logger.debug(f"HTTP Request Body: {response.request.body}")
-            self.logger.debug(f"HTTP Request Headers: {response.request.headers}")
+            self.logger.debug(
+                f"HTTP Request Headers: {response.request.headers}")
 
             self.logger.debug(f"HTTP Status Code: {response.status_code}")
             self.logger.debug(f"HTTP Response Headers: {response.headers}")
             self.logger.debug(f"HTTP Response Text: {response.text}")
 
             response.raise_for_status()
-            responses += response.json()
-        
+            findings = response.json()
+
+            for idx, d in enumerate(chunk):
+                for k, v in d.items():
```

Comment:
this does not quite work as we are not requiring ordered dicts, so this might be different from the first time

Reply:
We should be OK since we're requiring python 3.6 and above. According to the [python standard library](https://docs.python.org/3/library/stdtypes.html#dict). 

>  Changed in version 3.7: Dictionary order is guaranteed to be insertion order. This behavior was an implementation detail of CPython from 3.6.